share/lib/python/neuron/tests/test_vector.py

```diff
@@ -101,9 +101,59 @@ def testNumpyInteraction(self):
         except:
             pass
 
+    def testSlicing(self):
+        l = [i for i in range(10)]
+        v = h.Vector(l)
+        assert(list(v[2:6]) == l[2:6]), "v[2:6] Failed"
+        assert(list(v[-3:-1]) == l[-3:-1]), "v[-3:-1] Failed"
+        assert(list(v[::2]) == l[::2]), "v[::2] Failed"
+        assert(list(v[::-2]) == l[::-2]), "v[::-2] Failed"
+        assert(list(v[3:6:2]) == l[3:6:2]), "v[3:6] Failed"
+        assert(list(v[7:-9:-2]) == l[7:-9:-2]), "v[7:-9:-2] Failed"
+        assert(list(v[-1::-3]) == l[-1::-3]), "v[-1::-3] Failed"
+        assert(list(v[-2::-6]) == l[-2::-6]), "v[-2::-6] Failed"
+        assert(list(v[7:-1:-2]) == l[7:-1:-2]), "v[-2::-6] Failed"
+        assert(list(v[3:2:-2]) == l[3:2:-2]), "v[3:2:-2] Failed"
+
+    def testRandomSlicing(self):
```

Comment:
I think it's good to have done the random test, but I think we want deterministic tests in the repo